filament/src/RenderTarget.cpp

```diff
@@ -102,12 +102,14 @@ FRenderTarget::HwHandle FRenderTarget::createHandle(FEngine& engine, const Build
     // note that post-processing includes FXAA by default.
     const uint8_t samples = 1;
 
-    const TargetBufferInfo cinfo(upcast(color.texture)->getHwHandle(),
-            color.mipLevel, color.face);
+    const TargetBufferInfo cinfo =
+            upcast(color.texture)->getTarget() == Texture::Sampler::SAMPLER_CUBEMAP ?
+            TargetBufferInfo(upcast(color.texture)->getHwHandle(), color.mipLevel, color.face) :
+            TargetBufferInfo(upcast(color.texture)->getHwHandle(), color.mipLevel, color.layer);
 
     const TargetBufferInfo dinfo(
             depth.texture ? upcast(depth.texture)->getHwHandle() : TextureHandle(),
-            color.mipLevel, color.face);
+            color.mipLevel, color.layer);
```

Comment:
I think this should be `depth.mipLevel` and `color.layer`, no?

Reply:
Yes this must have been my typo.  😭

Reply:
Wait, wouldn't this need to also check the sampler type to use either .face or .layer?

Reply:
Hmm, yeah I think it should for completeness. My initial thought was that a depth cubemap doesn't make sense, but I suppose there are use-cases (like point light shadow maps).